### PR TITLE
Feature/python ci

### PR DESCRIPTION
**Pull request by migwellian**
Build the shadow jar and run the python tests on Travis and Appveyor

### CODE REVIEW

File: keanu-python/tests/test_appveyor_hanging.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import keanu as kn
+import numpy as np
+import sys
+import logging
+
+
+def test_appveyor_hanging():
```
Comment (GeorgeNash) [inline keanu-python/tests/test_appveyor_hanging.py]: Do we need this now that AppVeyor is no longer hanging - or do you want to keep it just incase it starts happening again?

File: keanu-python/tests/test_appveyor_hanging.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    print("test_appveyor_hanging")
+    logging.warn("test_appveyor_hanging")
```
Comment (GeorgeNash) [inline keanu-python/tests/test_appveyor_hanging.py]: If we are keeping this - can you swap this for `logging.warning` - it's chucking out some annoying deprecated messages when you build

File: .travis.yml [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
-  - 'if [[ ! -z $SONAR_TOKEN ]]; then sonar-scanner; fi'
+matrix:
+  include:
+    - language: java
+      jdk: oraclejdk8
+      addons:
+        sonarcloud:
+          organization: "improbable-keanu"
+      script:
+        - ./gradlew clean build -x :keanu-python:build -i
```
Comment (GeorgeNash) [inline .travis.yml]: Why do we need to run `keanu-python:build` here - aren't we running this again in the python section of the build?

File: keanu-python/tests/test_appveyor_hanging.py (hunk 1 above)
Comment (migwellian) [reply keanu-python/tests/test_appveyor_hanging.py]: Good call. I will remove it and we can always refer back to this PR if we need to reinstate it.

File: .travis.yml (hunk 3 above)
Comment (migwellian) [reply .travis.yml]: The `-x` means exclude

**Pull request merged**